src/main/java/spoon/support/template/SubstitutionVisitor.java

```diff
@@ -482,4 +488,18 @@ public void scan(CtElement element) {
 		} catch (DoNotFurtherTemplateThisElement ignore) {
 		}
 	}
+
+	public CtElement substitute(CtElement element) {
```

Comment:
yes, it is the entry point now. I forgot to document it. Thank You for noticing that!